web/src/components/header.js

```diff
@@ -27,14 +30,22 @@ const nav = [
     },
 ];
 
+const subnav = [
+    {
+        sublink: 'sublink',
+        subnav: '/hello',
+        sublinkcolor: 'brand-4',
+    },
+];
+
```

Comment:
@kcbreuil I would add these to the nav object. I'd also, using the designs, add the specific sub nav items that are called out.